src/components/BlockchainList/Post.tsx

```diff
@@ -74,10 +74,20 @@ export default function ({
   clickablePost?: boolean
   canReply?: boolean
 }) {
+  const [, setLocation] = useLocation()
+
   return (
     <div data-anchor={`#id=${blockchainId}`}>
       <Card hoverEffect={clickablePost}>
-        <Link to={`/thread/${blockchainId}`}>
+        <a
+          onClick={(e: MouseEvent) => {
+            if (e.target instanceof HTMLAnchorElement) {
```

Comment:
Lets remove curly braces

Reply:
Also, will the `button` work? Probably not, I already done this feature, I think it's better to wrap only upper part of card into Link OR use `clickable=true` on elements, that are clickable and check for this here. But if it works in this case, I guess it's fine